muser/pom.xml

```diff
@@ -0,0 +1,112 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please do a right click in IntelliJ and "Reformat code" for all files so they follow the IntellJ template.